### DISCUSSION TITLE
How to repair it

### DISCUSSION THREAD

**Discussion by tranquyen259** [James-Yu/LaTeX-Workshop#3109] [Q&A]
![image](https://user-images.githubusercontent.com/88999643/129511497-5c0373a7-f6c9-4869-b7a0-fec3d00a34db.png)

### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you can not see the requested feature. [Required]

You still see this issue?: Yes/No

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before requesting a feature.

You visited the wiki?: Yes/No

### If your requested feature is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes/No

## To Reproduce  [Required]

3. See error

## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
To access the log, click the 'TeX' icon on the Activity Bar on the left side, select 'View Log Messages', then select 'View LaTeX Workshop extension log'.

[Paste the log here]
```

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools' -> 'Console'. Paste anything suspicious.

[Paste the log here]
```

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: [e.g. Windows 10]
 - VS Code version: [e.g. 1.41.0]
 - Extension version: [e.g. 8.5.0]
 - TeX distribution version: [e.g. TeX Live 2019/MiKTeX 2.9.7250]

### Are you using VSCodium?

Yes/No

### Are you using the Snap or Flatpack versions of VS Code?

Yes/No

### Are you using LaTeX Workshop with VS Code Remote?

Yes/No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
